src/decorator.ts

```diff
@@ -38,6 +40,17 @@ export class Decorator {
         return this.autoFold
     }
 
+    toggleTailwindConfigFound(): void {
+        const filesNames = fs.readdirSync(this.currentWorkingFolder)
+
+        filesNames.map((file: string) => {
+            if (!(file === "tailwind.config.js")) {
```

Comment:
this doesn't seem to be a good way to identify tailwind projects. 
Since the config can also be called `tailwind.config.js|tailwind.config.ts|tailwind.config.cts|tailwind.config.mts|tailwind.config.cjs|tailwind.config.mjs` ...